# 时间戳
Header timeHeader
# 关节位置命令 I1 T2 T3 T4 I5
float32[] CommandPosData
# 关节速度命令
float32[] CommandVelData
